PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -2490,6 +2490,33 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return TRUE;
 		}
 
+		case NPPM_INTERNAL_RESTOREMONOINSTANCE:
+		{
+			// When mono instance, bring this one to front
+			if (nullptr != _pTrayIco && _pTrayIco->isInTray())

```

Comment:
Please use `_pTrayIco != nullptr !` instead of `nullptr != _pTrayIco`.

Reply:
Changed as per request.